mscore/playpanel.cpp

```diff
@@ -221,7 +221,7 @@ void PlayPanel::setEndpos(int val)
 void PlayPanel::setTempo(double val)
       {
       int tempo = lrint(val * 60.0);
-      tempoLabel->setText(QString("%1 BPM").arg(tempo, 3, 10, QLatin1Char(' ')));
+      tempoLabel->setText(QString(tr("Tempo") +"\n%1 " + tr("BPM")).arg(tempo, 3, 10, QLatin1Char(' ')));
```

Comment:
could well be `tr("Tempo\n%1 BPM").arg(tempo, 3, 10, QLatin1Char(' ')` too

Reply:
OK. Thanks. Changed and squashed.